.gitignore

```diff
@@ -1,2 +1,15 @@
 node_modules/
 .tern-port
+
+# =========================
+# Package manger lock file
+# =========================
+
+package-lock.json
+yarn.lock
```

Comment:
yarn.lock and package-lock.json *do* belong in Git-- at least one or the other. Distributing them insures that users install exactly the same set of dependencies that the maintainers tested with. This adds consistency which avoids strange bugs. So please remove this from the PR.

If you don't want them to show up as changed locally for you, you can use: https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

Please use amend and rebase to remove the change rather adding a new commit to the branch.

Reply:
Refinement: `yarn.lock` has been removed from the project, but `package-lock.json` is intended to be commited.